__mocks__/next/router.js

```diff
@@ -1,14 +1,15 @@
 const push = jest.fn()
+const back = jest.fn()
 const query = {}
 export const useRouter = jest.fn().mockImplementation(() => {
   return {
     route: '/',
     pathname: '/',
     query,
     asPath: '/',
-    push: push,
+    push,
     replace: async () => true,
     reload: () => null,
-    back: () => null
+    back
```

Comment:
This way you write less boilercode if you need to check parameters with which `push` was called (simple object destruction instead of adding spies every time). Same as with `push`.